material-popup-menu/src/main/res/values/styles.xml

```diff
@@ -7,6 +7,10 @@
         <item name="mpm_secondaryTextColor">@color/mpm_black_54_opacity</item>
         <item name="mpm_activeIconColor">@color/mpm_black_54_opacity</item>
         <item name="mpm_separatorColor">@color/mpm_black_12_opacity</item>
+        <item name="mpm_paddingBottom">@dimen/mpm_popup_menu_vertical_padding</item>
```

Comment:
Since this is the default and padding is now set programmatically in `MaterialRecyclerViewPopupWindow ` then the following lines can be removed from `mpm_popup_menu.xml`:
```
    android:paddingTop="@dimen/mpm_popup_menu_vertical_padding"
    android:paddingBottom="@dimen/mpm_popup_menu_vertical_padding"
```

Reply:
Done